File: python/01-commands/tests/MultiplesOf3or5/multiples_of_3_or_5.py
class MultiplesOf3or5:
    def __init__(self, number):
        self._number = number

    def sum(self):
        if self._number < 0:
            return 0

        sum = 0

        while self._number > 0:
            self._number -= 1
            if self._number % 3 == 0 or self._number % 5 == 0:
                sum += self._number

        return sum

File: dotnet/01-commands/tests/MultiplesOf3or5/MultiplesOf3or5.cs
namespace MultiplesOf3or5
{
    public class MultiplesOf3or5
    {
        private int _number;

        public MultiplesOf3or5(int number)
        {
            _number = number;
        }

        public int Sum()
        {
            if (_number < 0) return 0;

            int sum = 0;

            while (_number > 0)
            {
                _number--;
                if (_number % 3 == 0 || _number % 5 == 0)
                {
                    sum += _number;
                }
            }

            return sum;
        }
    }
}